workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/Readers/ReaderUpdateTests.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using Generated.Improbable.Gdk.Tests.BlittableTypes;
+using Improbable.Gdk.Core.MonoBehaviours;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Improbable.Gdk.Core.EditmodeTests.MonoBehaviours.Readers
+{
+    [TestFixture]
+    public class ReaderUpdateTests : ReaderWriterTestsBase
+    {
+        private void QueueUpdatesToEntity(params SpatialOSBlittableComponent.Update[] updatesToSend)
```

Comment:
This code appears to be duplicated with the non-blittable reader tests.

Can this be made generic and a shared test utility?